ethereum/permissioning/src/main/java/org/hyperledger/besu/ethereum/permissioning/NodeSmartContractV2PermissioningController.java

```diff
@@ -70,8 +70,8 @@ private boolean isPermitted(final EnodeURL enode) {
       LOG.trace("Permitted? {} for DNS {}", isIpToDNSEnodePermitted, ipToDNSEnode);
       return isIpToDNSEnodePermitted;
     } catch (final IllegalStateException illegalStateException) {
-      LOG.info("Unable to check permissions for enode {} ", enode, illegalStateException);
-      return false;
+      throw new IllegalStateException(
```

Comment:
Where is this caught and what changes now that we are throwing that exception?

Reply:
If this happens in the validation check at startup, it is caught by picoCLI and startup is aborted. This line change actually reverts to previous state. As in, we've changed this part back to how it was before #3697